docs_src/vision.learner.ipynb

```diff
@@ -514,8 +514,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Can you just put backticks around the argument names? For instance `k`, `heatmap`. Then same for the functions `plot_top_losses` (without parenthesis) will then create a link to that function documentation during our conversion, so it's pretty neat.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/1640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>